cmd/cloud/cluster.go

```diff
@@ -14,9 +14,11 @@ func init() {
 	clusterCreateCmd.Flags().String("provider", "aws", "Cloud provider hosting the cluster.")
 	clusterCreateCmd.Flags().String("size", "SizeAlef500", "The size constant describing the cluster.")
 	clusterCreateCmd.Flags().String("zones", "us-east-1a", "The zones where the cluster will be deployed. Use commas to separate multiple zones.")
+	clusterCreateCmd.Flags().Bool("cluster-ready", true, "Controls if the command should wait for k8s to become fully ready before exiting")
```

Comment:
Nit: missing trailing period

Reply:
Also, would it be simpler if we just called this `--wait`? In the future, we'll likely be invoking the REST API (like mmctl), and waiting will have multiple meanings.